src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -174,20 +182,21 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
     }
 
     private getColors(color:string) {
```

Comment:
the naming here is very bad.. you're getting a _single_ color, and the parameter is the alteration type not the color...